includes/admin/models/class-wppr-review-model.php

```diff
@@ -706,6 +706,16 @@ public function get_name() {
 		return apply_filters( 'wppr_name', $this->name, $this->ID, $this );
 	}
 
+	/**
+	 * Filter to display review product name in comparison table
+	 *
+	 * @since   3.4.3
+	 * @access  public
+	 * @return bool
+	 */
+	public function wppr_hide_name() {
+		return apply_filters( 'wppr_hide_product_name', true );
+	}
```

Comment:
@UVLabs adding name, ID and $this like on line 706 would make this filter more useful

Reply:
@contactashish13 not sure what you mean here, I'm using this filter as a bool to simply check whether to display the review name or not in the comparison table: 

https://github.com/UVLabs/wp-product-review-pro/blob/3a10061e1fd1048b3646ba21a8f11a32cd9b6091/includes/addons/layouts/table.php#L50-L55

Reply:
@UVLabs filters are for others, not for us :) consider a user who wants to extend the filter and wants to return false when the name of the product is "apple". Can he do that with your current filter? The more arguments, the better.

Reply:
@contactashish13 ok thanks

Reply:
@UVLabs code example of what I, as a user, might want to do. 

```
add_filters( 'wppr_hide_product_name', 'xxx', 10, 3);
function xxx($show, $id, $model){
// hide name for all products that do not have apple in their name.
return strpos($model->get_name(), 'apple') !== false);
}
```
By giving me absolutely no parameters, you are making the filter useless for me to use. Its a simple thumb rule. If you dont want to provide parameters, why have a filter at all? A filter is a function and a function without parameters is useless.

Reply:
@UVLabs, to be honest, I don't understand the role of this method at all, if you just want to allow a way of hiding the review title in the comparison table, why you don't use the filter directly in the comparison table code view. 
This is more template related method, not something that has to do with the review proprieties.